README.md

```diff
@@ -1,2 +1,19 @@
-# nature_news_disparities
-Analysis pipeline for Nature news articles
+# Analysis Pipeline for Nature News Articles
+
+Note: This is currently a work in progress -- many things will change and may also be undocumented at this moment.
+
+The code is made of 2 parts currently: 
+1. Scrape Nature News Articles
+2. Analyze the text of the articles to identify possible speaker biases
+
+## Scraping
+  
+The code relies upon [scrapy](https://docs.scrapy.org/en/latest/index.html) to crawl links and process the articles.
```

Comment:
Really cool!! Didn't know this was out there. Definitely looks like a cool package to use.

Reply:
yeah! I found it easy to use and it's well documented, (my husband pointed it out to me). I am a super novice in scraping so I needed something that was gonna almost do it all for me.